minimisers/DREAM/functions/makeEmptyBayesResultsStruct.m

```diff
@@ -181,21 +252,40 @@
 allChains = [1 1 1];
 coder.varsize('allChains',[1e4 50 50],[1 1 1]);
 
+% Nested Sampler
+% result.logX = logZ;
+% result.nest_samples = nest_samples;
+% result.post_samples = post_samples;
+LogZ = 0;
+
+nestSamples = [0 0];
```

Comment:
Since this file contains nested sampler result, would it be better to put this in the generalUtils folder instead?

Reply:
Yes